examples/paddle/train.py

```diff
@@ -0,0 +1,96 @@
+import mlflow.paddle
+import paddle
+from paddle.nn import Linear
+import paddle.nn.functional as F
+import numpy as np
+import os
+import random
+from pathlib import Path
+import wget
+
+def load_data():
+    url = 'https://archive.ics.uci.edu/ml/machine-learning-databases/housing/housing.data'
+
+    datafile = Path('housing.data')
+    if not datafile.exists():
+        datafile = wget.download(url)
+
+    data = np.fromfile(datafile, sep=' ', dtype=np.float32)
+    feature_names = [ 'CRIM', 'ZN', 'INDUS', 'CHAS', 'NOX', 'RM', 'AGE', \
+                      'DIS', 'RAD', 'TAX', 'PTRATIO', 'B', 'LSTAT', 'MEDV' ]
+    feature_num = len(feature_names)
+    data = data.reshape([data.shape[0] // feature_num, feature_num])
+    ratio = 0.8
+    offset = int(data.shape[0] * ratio)
+    training_data = data[:offset]
+
+    maximums, minimums, avgs = training_data.max(axis=0), training_data.min(axis=0), \
+                                 training_data.sum(axis=0) / training_data.shape[0]
```

Comment:
```suggestion
    maximums = training_data.max(axis=0)
    minimums training_data.min(axis=0)
    avgs = training_data.sum(axis=0) / training_data.shape[0]
```

Can we split this line?